lib/routemap_northbound.c

```diff
@@ -1197,6 +1221,12 @@ const struct frr_yang_module_info frr_route_map_info = {
 				.destroy = lib_route_map_destroy,
 			}
 		},
+		{
+			.xpath = "/frr-route-map:lib/route-map/optimization-disabled",
+			.cbs = {
+				.modify = lib_route_map_optimization_disabled_modify,
```

Comment:
Missing `cli_show` callback?

Reply:
This command is/was never shown in the config.

Reply:
@rwestphal Actually I think maybe we should create a new command like "route-map NAME optimization" and show it in the config. And mark the old one (from within the RMAP_NODE) as deprecated and remove it eventually. Because right now it is not possible to make this configuration parameter stay after the reboot. What do you think?

Reply:
Yeah, I think that's the ideal thing to do. I'm glad you already updated this PR implementing that logic.